recipes/yodapy/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.2.3b0" %}
+
+package:
+  name: yodapy
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/y/yodapy/yodapy-{{ version }}.tar.gz
+  sha256: 48052e96bdfa1e065d18a3bd91c73100d789e35b8f37c585e7914e1403b5e1ab
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python ==3.6
```

Comment:
Are there any known incompatibilities with Python 3.7?

Reply:
I had issues with the pinned gevent. It doesn't seem to support 3.7 yet. In future release of yodapy, I will try to solve this issue. Thanks! :D